tests/table/test_init.py

```diff
@@ -310,6 +310,19 @@ def test_table_scan_row_filter(table_v2: Table) -> None:
     assert scan.filter(EqualTo("x", 10)).filter(In("y", (10, 11))).row_filter == And(EqualTo("x", 10), In("y", (10, 11)))
 
 
+def test_table_scan_partition_filters_case_sensitive(table_v2: Table) -> None:
```

Comment:
nit: the goal of this test is to show that case sensitive scan works. i think a more readable test can be something like
1. assert data exists for `row_filter=EqualTo("X", 10)`
2. assert case sensitive scan for `row_filter=EqualTo("X", 10)` returns the same data
3. assert case insensitive scan for `row_filter=EqualTo("X", 10)` and `row_filter=EqualTo("x", 10)` also returns the same data

Reply:
Thanks very much @kevinjqliu , I've updated the test case for scan and the other operations as well. Hope it looks better now?